widgetssdk/src/main/AndroidManifest.xml

```diff
@@ -68,14 +68,9 @@
             android:name=".callvisualizer.EndScreenSharingActivity"
             android:launchMode="singleTask"
             android:theme="@style/Application.Glia.Call.Activity" />
-        <activity
-            android:name=".permissions.PermissionsSupportActivity"
-            android:launchMode="singleTask"
-            android:noHistory="true"
-            android:theme="@style/Application.Glia.Translucent.Activity" />
 
         <activity
-            android:name=".view.dialog.holder.DialogHolderActivity"
```

Comment:
To be honest, I like the `DialogHolderActivity` name more. To me, `GhostActivity` is associated with some spy activity and may sound dangerous for integrators.

Reply:
Makes sense. Will change it back

Reply:
fixed